src/_sass/color_schemes/centrapay-theme.scss

```diff
@@ -121,18 +120,26 @@ $turquoise-700: #107D75;
 $turquoise-800: #12615B;
 $turquoise-900: #114E4A;
 
-// $sidebar-color: $neutral-900;
+// Generic theme overrides. Browse how they're used with:
+// bundle open just-the-docs
 $sidebar-color: $chocolate-050;
 $link-color: $chocolate-600;
 $nav-child-link-color: $neutral-800;
-
 $body-background-color: $white;
 $search-background-color: $white;
-$border-color: $chocolate-200;
+$border-color: $chocolate-100;
```

Comment:
Not strictly code blocks, this line just made the borders on tables look less strange.